api/license.yaml

```diff
@@ -4,6 +4,19 @@ group:
   name: license
   description: OEE license management
 commands:
+  - name: info
+    description: Get license info
+    descriptionLong: Get license status and expiration date
+    method: GET
+    path: service/apama-oeeapp/mon/info/license
```

Comment:
- I wonder if we could hide this API call? It is not really useful and could be confused by the user with the `license_info` command
- or we replace the call to `c8y oee license info` in line 101 with a direct call to `service/apama-oeeapp/mon/info/license` ?

Reply:
it is already set to hidden: true so it wont be shown on the cli console.